eng/common/tools.ps1

```diff
@@ -11,6 +11,12 @@
 # Binary log must be enabled on CI.
 [bool]$binaryLog = if (Test-Path variable:binaryLog) { $binaryLog } else { $ci }
 
+# Set to true to use the pipelines logger which will enable Azure logging output.
+# https://github.com/Microsoft/azure-pipelines-tasks/blob/master/docs/authoring/commands.md
+# This flag is meant as a temporary opt-opt for the feature while validate it across
+# our consumers. It will be deleted in the future.
+[bool]$pipelinesLog = if (Test-Path variable:pipelinesLog) { $pipelinesLog } else { $ci }
```

Comment:
The most suspicious change in Arcade is the addition of a new logger. We've seen lots of perf problems in the past with MSBuild loggers that are not implemented correctly. 

@ChadNedzlek - how can I set `variable:pipelinesLog` to `false` to disable the new logger to check if this is the problem?

Reply:
I decided to edit this file directly to test my theory. By setting `$pipelinesLog = $false` on the next line, build time went back to the normal value <10 minutes as opposed to timing out after 60. 

Seems like there is something very wrong with this new logger.